whipper/common/mbngs.py

```diff
@@ -153,18 +153,80 @@ def getIds(self):
 
 
 def _getWorks(recording):
-    """Get "performance of" works out of a recording."""
+    """
+    Get 'performance of' works out of a recording.
+
+   :param recording: recording entity in MusicBrainz
+   :type recording: dict
+   :returns: list of works being a performance of a recording
+   :rtype: list
+    """
     works = []
-    valid_work_rel_types = [
-        'a3005666-a872-32c3-ad06-98af558e99b0',  # "Performance"
-    ]
+    valid_type_id = 'a3005666-a872-32c3-ad06-98af558e99b0'  # "Performance"
     if 'work-relation-list' in recording:
         for work in recording['work-relation-list']:
-            if work['type-id'] in valid_work_rel_types:
-                works.append(work['work']['id'])
+            if work['type-id'] == valid_type_id:
+                works.append(work['work'])
     return works
 
 
+def _getWorksId(works):
```

Comment:
It just came to my mind that this being a separate method isn't that useful.
It can also be replaced with a single line (set comprehension):

```python
sorted({work['id'] for work in works})
```

Reply:
Done!